cmd/gvisor-website/main.go

```diff
@@ -41,6 +41,9 @@ var redirects = map[string]string{
 
 	// Deprecated, but links continue to work.
 	"/cl": "https://gvisor-review.googlesource.com",
+	// Redirect for old url
+	"/docs/user_guide/compatibility/amd64":  "/c/linux/amd64",
```

Comment:
Redirect to "/c/linux/amd64/" (slash at the end)?

Reply:
I was redirecting to a redirect helper URL so it the slash didn't matter in that case. I instead updated to the final URL to avoid more redirects. Also updated other redirects to end in slashes.